### DISCUSSION TITLE
Disable MFA from server

### DISCUSSION THREAD

**Discussion by tibolam** [snipe/snipe-it#12304] [General]
Hello All

I just replaced my phone and I didn't transfer the Google Authenticator. Now I can log in to snipe-it without the code.  I'm the admin; therefore, I can reset from the portal.  Is there a way that we can reset/remove it from the server?  

I have Windows Server environment.

Thanks.
Nam

**Comment by corydlamb** [comment]
Saw this in a thread thought this command would be useful for you. Make sure the password is in accordance to your password policy and you should be able to use that account to go to users and disable your account's 2fa.

php artisan snipeit:create-admin --first_name=ADMINISTRATOR --last_name=SUPER --email=[helpdesk@mydomain.com](mailto:helpdesk@mydomain.com) --username=administrator --password=password123
